src/style-dictionary-dist2/TerrawareTheme2.ts

```diff
@@ -459,7 +459,7 @@ const TerrawareTheme2: ThemeDictionary = {
     "TwClrBgTertiaryActive": "#B2AB93",
     "TwClrBgTertiaryHover": "linear-gradient(rgb(178 171 147 / 50%), rgb(178 171 147 / 50%)), linear-gradient(#CBC6B7, #CBC6B7)",
     "TwClrBrdr": "#333025",
-    "TwClrBrdrActive": "linear-gradient(rgb(70 88 170 / 50%), rgb(70 88 170 / 50%)), linear-gradient(#6172BE, #6172BE)",
+    "TwClrBrdrActive": "mix(#4658AA, #6172BE, 50%)",
```

Comment:
@constanzauanini was this a manual edit or did you re-run `yarn build-dictionary`?

Reply:
I ran yarn build dictionary, because @chudo76 updated the token

Reply:
@chudo76 this would have been a good use-case to run the design changes workflow instead :)

Reply:
Ah yes, good to know. For this one in particular, I wasn't confident about the change since it was our first time using the `mix()` Sass mixin.

Reply:
Got it, yes good point..